### PR TITLE
New calendar area

### PR DESCRIPTION
**Pull request by jamesalmond**

### CODE REVIEW

File: app/controllers/admin/calendar_areas_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    end
+
+    def edit
+    end
+
+    def update
+      if @calendar_area.update(calendar_area_params)
+        redirect_to admin_calendar_areas_path, notice: 'Calendar area updated'
+      else
+        render :edit
```
Comment (jamesjefferies) [inline app/controllers/admin/calendar_areas_controller.rb]: it's always the edits which are lonely and untested ;)

File: lib/energy_sparks/csv_loader.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require 'csv'
+module EnergySparks
+  class CsvLoader
+    def self.default_options
```
Comment (jamesjefferies) [inline lib/energy_sparks/csv_loader.rb]: Now we have set this up, at some point I guess we have some other CSV parsers which we should switch to using this....

File: app/controllers/admin/calendar_areas_controller.rb (hunk 1 above)
Comment (jamesalmond) [reply app/controllers/admin/calendar_areas_controller.rb]: I'll add a little tweak to the test to make sure it's not all red and embarassed

File: lib/energy_sparks/csv_loader.rb (hunk 2 above)
Comment (jamesalmond) [reply lib/energy_sparks/csv_loader.rb]: Yes, as we come across them I thought we could move them across.

**Pull request merged**